assayist/processor/utils.py

```diff
@@ -137,12 +137,65 @@ def unpack_rpm(rpm_file, output_dir):
     log.info(f'Successfully unpacked {os.path.split(rpm_file)[-1]} to {output_dir}')
 
 
+def unpack_container_image(container_image_file, output_dir):
+    """
+    Unpack a container image to the specified directory
+
+    :param str container_image_file: the path to the container image file to unpack
+    :param str output_dir: the path to unpack the container image to
+    """
+    # Unpack the manifest.json file from which we figure out the latest image layer
+    tar = tarfile.open(container_image_file)
+    manifest_file = tar.extractfile('manifest.json')
+    manifest_data = json.loads(manifest_file.read().decode('utf-8'))
+    layer_to_unpack = manifest_data[0]['Layers'][-1]
+
+    # Unpack the last layer, which itself is a .tar file
+    tar.extract(layer_to_unpack)
+    tar.close()
+
+    # Extract the file system contents from the last layer
+    with tarfile.open(layer_to_unpack) as tar:
+        tar.extractall(output_dir)
+
+    # Remove extracted layer .tar file
+    shutil.rmtree(os.path.split(layer_to_unpack)[0])
```

Comment:
Confused about why this is more than just `os.remove()`. Isn't it just a tar file?

Reply:
The individual `layer.tar` files are stored in dirs:

```bash
├── c4311fef506a3e9e4134be58a18fe10095badcfa17c7f7b94f1ff4c210764f85
│   ├── json
│   ├── layer.tar
│   └── VERSION
├── e1a24501757d9e59a711c20987e0a1f3d88c49aa315a1ad6baa45c682354110c
│   ├── json
│   ├── layer.tar
│   └── VERSION
```

So when you untar just the specific `layer.tar`, it includes its directory. So when deleting, we want to delete the `e1a24501757d9e59a711c20987e0a1f3d88c49aa315a1ad6baa45c682354110c` directory as well as the `layer.tar` file.